conda/environment-release-build.yml

```diff
@@ -4,18 +4,6 @@ channels:
 dependencies:
   - python=3.8
 
-  # runtime
-  - contourpy >=1
-  - jinja2 >=2.7
-  - numpy >=1.11.3
-  - packaging >=16.8
-  - pillow >=4.0
-  - python-dateutil >=2.1
-  - pyyaml >=3.10
-  - tornado >=5
-  - typing_extensions >=3.10.0
-  - xyzservices>=2021.09.1
-
```

Comment:
These should no longer be needed given out isolate setuptools builds, so should be removed as redundant